jenkins/shared-libraries/linux/vars/ExecuteCodeQL.groovy

```diff
@@ -47,25 +47,29 @@ def call(org, repo, branch, language, buildCommand, token, installCodeQL) {
     }
 
     sh '''
+        echo "WORKSPACE: ${WORKSPACE}"
```

Comment:
In a follow on PR can we update this to be natural language so users don’t send us requests trying to guess what these values mean:

```sh
echo “Current Jenkins Workspace: ${WORKSPACE}”
echo “Current working directory: ${PWD}”
echo “CodeQL database path: ${DATABASE_PATH}”
echo “CodeQL database bundle file name: ${DATABASE_BUNDLE}”
```